Index: src/database/migration/1648750641709-CreateSurveysUsers.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/database/migration/1648750641709-CreateSurveysUsers.ts b/src/database/migration/1648750641709-CreateSurveysUsers.ts
new file mode 100644
--- /dev/null	(date 1648751090053)
+++ b/src/database/migration/1648750641709-CreateSurveysUsers.ts	(date 1648751090053)
@@ -0,0 +1,42 @@
+import {MigrationInterface, QueryRunner, Table} from "typeorm";
+
+export class CreateSurveysUsers1648750641709 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.createTable(
+            new Table({
+                name:'surveys_users',
+                columns:[
+                    {
+                        name:'id',
+                        type:'uuid',
+                        isPrimary: true
+                    },
+                    {
+                        name:'user_id',
+                        type:'uuid'
+                    },
+                    {
+                        name:'survey_id',
+                        type:'uuid'
+                    },
+                    {
+                        name:'value',
+                        type:'number',
+                        isNullable:true
+                    },
+                    {
+                        name:'created_at',
+                        type:'timestamp',
+                        default:'now()'
+                    }
+                ]
+            })
+        )
+    }
+
+    public async down(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.dropTable('surveys_users')
+    }
+
+}
